tools/create-delete-namespaces.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Copyright 2022 Hewlett Packard Enterprise Development LP
+# Other additional copyright holders may be indicated within.
+#
+# The entirety of this work is licensed under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+usage() {
+    cat <<EOF
+Cycle through creation and deletion of namespaces
+Usage: $0 [NAMESPACE-SIZE-IN-BYTES]
+
+NAMESPACE-SIZE-IN-BYTES: 0 - Entire drive
+EOF
+}
+
+NVME_SPACE_SIZE=$1
```

Comment:
do you mean NAMESPACE_SIZE? 

To be consistent with nvme.sh I'd make this and the usage() be `SIZE` and default it to zero `${1:-0}`